### DISCUSSION TITLE
Update on deprecation of Keycloak adapters

### DISCUSSION THREAD

**Discussion by stianst** [keycloak/keycloak#19417] [Adapter deprecation]
After a lot of feedback we have decided to revisit our plans around adapters. We will no longer remove the adapters in Keycloak 19, well that ship sailed a while ago, or in the near future, but are delaying these plans until we have found alternative solutions. We do still plan to eventually remove the majority of adapters though in the long run.

For more information take a look at https://www.keycloak.org/2023/03/adapter-deprecation-update.html.

**Comment by kkcmadhu** [comment]
thanks for keeping the support for adapters @stianst , it is really a life saver for us!
